cartridges/int_cloudinary/cartridge/scripts/jobs/steps/cldUpdateProductImagesForFeeds.js

```diff
@@ -83,7 +84,7 @@ function writeProductFileContent(cloudinaryUrlStreamWriter, productSearchHitsItr
             if (!empty(imgAssetsSorted) && !empty(imgAssetsSorted.resources) && imgAssetsSorted.resources.length > 0) {
                 if (!empty(params.jobLastExecutionTime)) {
                     skipCurrentProduct = true;
-                    for (var i = 0; i < imgAssetsSorted.resources.length; i++) {
+                    for (let assetsSorted of imgAssetsSorted.resources) {
                         productDate = new Date(imgAssetsSorted.updatedAt);
```

Comment:
Should it be assetsSorted.updateAt?  Otherwise, the loop doesn't have sense

Reply:
changes are done

Reply:
I'm not so sure - imgAssetsSorted.resources includes several resources so what does it mean "imgAssetsSorted.updatedAt"? We would like to check the updatedAt of each specific resource

Reply:
we don't have updatedAt property in "imgAssetsSorted.resources"

Reply:
we don't have "updatedAt" property in "assetsSorted" element